components/form/Input.js

```diff
@@ -2,11 +2,10 @@ import React, { useState } from 'react';
 import { useFormContext } from 'react-hook-form';
 import PropTypes from 'prop-types';
 import { ShowPasswordIcon, HidePasswordIcon } from '@/public/assets/signUp/passwordSvgs';
-import FormIcons from '@/components/form/FormIcons';
 import { Label, ErrorMessage } from '.';
 import classNames from 'classnames';
 import { showError, showSuccess } from '@/utils/form-helpers';
-import { ToolTipIcon } from '@/public/assets/dashboard/navBarIcon';
+import { HelpIcon, SuccessIcon } from '@/public/assets/dashboard/navBarIcon';
```

Comment:
HelpIcon and SuccesIcon shouldn't be important from navBarIcon. There are no correlation between them, so we might create another file for them.